src/cache/cache_las.c

```diff
@@ -161,7 +163,7 @@ __wt_las_cursor_create(WT_SESSION_IMPL *session, WT_CURSOR **cursorp)
 	 * opens (the first update is safe because it's single-threaded from
 	 * wiredtiger_open).
 	 */
-	btree = S2BT(session);
+	btree = ((WT_CURSOR_BTREE *)(*cursorp))->btree;
```

Comment:
@agorrod:
> This change appears a few times, is it a consequence of no longer saving and restoring the session_dhandle in __meta_btree_apply? I'd rather keep the S2BT semantic if possible.

The locations where that happens are when opening cursors on the LAS file, the metadata file and the LSM primary chunk. I'm not sure about the LSM primary chunk, but the other two are deliberately not setting the WT_SESSION handle (although I didn't trace back why that's the case).

Anyway, to make S2BT work, we'd have to save/restore the WT_SESSION handle around the S2BT macro call, which seems like more work than it's worth to me.